docs/operations/single-server.md

```diff
@@ -42,37 +43,43 @@ The startup scripts for these example configurations run a single ZK instance al
 
 The example configurations run the Druid Coordinator and Overlord together in a single process using the optional configuration `druid.coordinator.asOverlord.enabled=true`, described in the [Coordinator configuration documentation](../configuration/index.md#coordinator-operation).
 
+The `start-druid` is a generic launch script for starting druid services on single server, it accepts optional arguments like services, memory and config.
+All reference configurations can be acheived by passing appropriate arguments to this script.
+Existing launch scripts are deprecated and will be removed in the next release. 
+
+
 While example configurations are provided for very large single machines, at higher scales we recommend running Druid in a [clustered deployment](../tutorials/cluster.md), for fault-tolerance and reduced resource contention.
 
 ## Single server reference configurations
 
-### Nano-Quickstart: 1 CPU, 4GiB RAM
+### Nano: 1 CPU, 4GiB RAM
 
-- Launch command: `bin/start-nano-quickstart`
-- Configuration directory: `conf/druid/single-server/nano-quickstart`
+- Launch command: `bin/start-druid --memory=4g`
+- Configuration directory: `conf/druid/single-server/quickstart`
```

Comment:
do we still need to list out the `Configuration directory:` is it is the same in every instance?

Reply:
No, we shouldn't have to specify the configuration directory as it is the same for all these profiles.

Reply:
I adhered to the existing format, but this is redundant information. I will remove it.